src/azul/service/manifest_service.py

```diff
@@ -1395,9 +1395,9 @@ def command_lines(cls,
             '--location',
             '--fail',
         ]
+        # These options are added to the command-line instead of being placed
+        # inside the curl manifest so the user is able to customize the values.
```

Comment:
```suggestion
        # Some options are added to the command-line instead of the curl manifest so that the user can more easily customize them.
```